src/ui/components/results/paginationcomponent.js

```diff
@@ -239,27 +253,32 @@ export default class PaginationComponent extends Component {
     }
 
     return {
-      ellipses: {
+      pinnedNumbers: {
         mobileBack: this._pinFirstAndLastPage && mobileBackLimit > 0,
```

Comment:
If the backLimit is 1, will this show both Page '1' twice?

Reply:
It will not.